docs/tutorials/enterprise_beginner/1_get_a_starter_website.md

```diff
@@ -1,10 +1,10 @@
-# Step 1 — Get a starter website [[% include 'snippets/experience_badge.md' %]]
+# Step 1 — Get a starter website [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]
 
 To set up the starter website, you need to follow these steps:
 
-## Get a clean [[= product_name_exp =]] installation
+## Get a clean [[= product_name =]] installation
 
-To begin the tutorial, you need a clean installation of [[= product_name_exp =]].
+To begin the tutorial, you need a clean installation of [[= product_name_exp =]] or [[= product_name_com =]].
```

Comment:
Using Commerce in the tutorial will probably backfire for beginner users, due to different front page, let's not suggest this.